### DISCUSSION TITLE
Feature votes and lifetime question

### DISCUSSION THREAD

**Discussion by Hr46ph** [bitgapp/eqMac#711] [General]
Hello,

As someone with a partial hearing impairment (right side -6dB dip around 4K) I am very much looking forward to the upcoming feature to separately adjust the left and right channels.

Is there any approximate timeline when you expect to include the feature? Same Q for the lifetime purchase.

Thanks!

**Comment by Hr46ph** [comment]
Is it a difficult question to answer?

I just like to know if its coming and possibly an approximate timeframe, like this year, early/late next year?

Not asking for promises or commitment, just what you expect.

**Comment by nodeful** [comment]
Hey @Hr46ph not difficult to answer, just have absolutely no time to chase down every question and answer it 😅 
Lifetime License coming within a week!
As for Separate L/R channel EQ I hope I will make it into the next Minor update v1.8 or 1.9, depends on how crazy it's going to get.
Probably looking at Feb 2023 time. Sorry for the wait!

**Reply by Hr46ph** [reply]
Better later than never! 😃 

Thanks, sounds awesome! (pun intented).

**Comment by Hr46ph** [comment]
Lifetime came and I picked it up. High hopes for Separate L/R channel EQ this year! 😃
